src/Hazelcast.Net.Tests/Clustering/MemberPartitionGroupServerTests2.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) 2008-2024, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Hazelcast.Networking;
+using Hazelcast.Testing;
+using Hazelcast.Testing.Conditions;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+namespace Hazelcast.Tests.Clustering
+{
+    [Category("enterprise")]
+    [ServerCondition("5.5")]
+    [Explicit("This test is not working as expected. It is failing on the GitHub Actions due to resource consumption.")]
+    [Timeout(60_000)]
+    public class MemberPartitionGroupServerTests2 : MultiMembersRemoteTestBase
```

Comment:
Is is standard practice to have `Class2` style naming in the C# codebase? This is not standard in Java

Reply:
Yeah, we have such separations on the repo. The test behaves flaky as I described above. I've separated the test suit but nothing change it. For now, I'll leave it as it is, and will run such tests on nightly runs. However, it requires effort because there is no nightly category. We just run all tests each time either daily or on PR builder.

Reply:
For better understanding, renamed the test.